### ISSUE TITLE
Areal output in MzMine not corresponding to that of Chemstation (area of base peak intensity rather than TIC?)

### ISSUE THREAD

**Issue by Chemecol** [mzmine/mzmine2#952]
### Basic information

* My operating system: Windows 10 (64)
* My MZmine version: 2.53

### What happened

I have some issues getting the height and the area of the peaks in my GC-MS data that matches with the data from chemstation. I manage to get an output in by following ADAP user manual page 18-31 [https://mzmine.github.io/ADAP_user_manual.pdf](url). However, it seems that the height of the peak and the area under the peak is not matching that of the data from chemstation. It's even far off so it doesn't have anything to do with the integration as far as my understanding goes. The thing is that base peak intensity seems to be the standard in MzMine, I want to have the height of the peak and the area of the TIC. Although I think I somehow get the height and area of the base peak intensity. Does anyone know how to manage this?

Below you can see the parameters used and the workflow.

![image](https://user-images.githubusercontent.com/95356124/144665930-ec861325-3c4f-4c59-b859-89cdd3921ef9.png)
Raw data methods -> Feature detection -> Mass detection (1st step)

![image](https://user-images.githubusercontent.com/95356124/144666126-632ed309-485e-4dbb-b4e9-f613d4cf3a20.png)
Raw data methods -> Feature detection -> ADAP Chromatogram builder (2nd step)

![image](https://user-images.githubusercontent.com/95356124/144666312-5ea93892-e3b3-45bc-92c4-b7f20f06767e.png)
Feature list methods -> feature detection -> Chromatogram deconvolution  (3rd step)

![image](https://user-images.githubusercontent.com/95356124/144667051-b74c2a3f-0484-401a-aaa1-28c7fedbb4c1.png)
Feature list methods -> Spectral deconvolution -> Hierarchal clustering (4th step)

![image](https://user-images.githubusercontent.com/95356124/144667280-848e323f-13f2-4ba2-816e-1421533ff0c9.png)
features list methods -> alignment -> ADAP alligner GC (5th step)

![image](https://user-images.githubusercontent.com/95356124/144667626-b0cde739-22be-4b9a-b91e-b7a6ade70de9.png)
Feature list methods -> Export/Import -> Export to CSV / Export to MSP file (ADAP) (6th step) 

![image](https://user-images.githubusercontent.com/95356124/144668388-3132d545-a201-4275-a3dd-1a80e21bd812.png)
Example of the output from MzMine and Chemstation on the same dataset, same peaks are colour coded.

Best,

Chemecol

**Comment by tomas-pluskal** [comment]
You are correct, MZmine always calculates the area from the trace of a single ion - which absolutely makes sense, if you think about it. I guess Chemstation was developed for low-resolution MS data, where it is not possible to trace a single ion, and that's why it is calculating the area from the TIC. MZmine was developed with the main focus on high-resolution data, so it doesn't offer this option.
